crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -1,3 +1,5 @@
+use std::collections::HashSet;
```

Comment:
I am using HashSet because it's faster considering there's the possibility of gaps between names. e.g. `account-1`, `account-2`, `account-3`. This way, we don't need to first sort if we were to use an array (`Vec`)